src/main/java/nl/knaw/dans/ttv/core/TransferItem.java

```diff
@@ -0,0 +1,347 @@
+    }
+
+    public void setAipTarEntryName(String aipTarEntryName) {
+        this.aipTarEntryName = aipTarEntryName;
+    }
+
+    public String getAipsTar() {
+        return aipsTar;
+    }
+
+    public void setAipsTar(String aipsTar) {
+        this.aipsTar = aipsTar;
+    }
+
+    public LocalDateTime getBagDepositDate() {
+        return bagDepositDate;
+    }
+
+    public void setBagDepositDate(LocalDateTime bagDepositDate) {
+        this.bagDepositDate = bagDepositDate;
+    }
+
+    @Override
+    public String toString() {
+        return "TransferItem{" +
+                "id=" + id +
+                ", datasetPid='" + datasetPid + '\'' +
+                ", datasetVersion='" + datasetVersion + '\'' +
+                ", versionMajor=" + versionMajor +
+                ", versionMinor=" + versionMinor +
+                ", creationTime=" + creationTime +
+                ", metadataFile='" + metadataFile + '\'' +
+                ", bagId='" + bagId + '\'' +
+                ", nbn='" + nbn + '\'' +
+                ", otherId='" + otherId + '\'' +
+                ", otherIdVersion='" + otherIdVersion + '\'' +
+                ", swordToken='" + swordToken + '\'' +
+                ", datasetDvInstance='" + datasetDvInstance + '\'' +
+                ", bagChecksum='" + bagChecksum + '\'' +
+                ", queueDate=" + queueDate +
+                ", bagSize=" + bagSize +
+                ", transferStatus=" + transferStatus +
+                ", oaiOre=" + Arrays.toString(oaiOre) +
+                ", pidMapping=" + Arrays.toString(pidMapping) +
+                ", aipTarEntryName='" + aipTarEntryName + '\'' +
+                ", aipsTar='" + aipsTar + '\'' +
+                ", bagDepositDate=" + bagDepositDate +
+                '}';
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        TransferItem that = (TransferItem) o;
+        return versionMajor == that.versionMajor && versionMinor == that.versionMinor && bagSize == that.bagSize && datasetPid.equals(that.datasetPid) && Objects.equals(datasetVersion, that.datasetVersion) && creationTime.equals(that.creationTime) && metadataFile.equals(that.metadataFile) && Objects.equals(bagId, that.bagId) && Objects.equals(nbn, that.nbn) && Objects.equals(otherId, that.otherId) && Objects.equals(otherIdVersion, that.otherIdVersion) && Objects.equals(swordToken, that.swordToken) && Objects.equals(datasetDvInstance, that.datasetDvInstance) && Objects.equals(bagChecksum, that.bagChecksum) && Objects.equals(queueDate, that.queueDate) && transferStatus == that.transferStatus && Arrays.equals(oaiOre, that.oaiOre) && Arrays.equals(pidMapping, that.pidMapping) && Objects.equals(aipTarEntryName, that.aipTarEntryName) && Objects.equals(aipsTar, that.aipsTar) && Objects.equals(bagDepositDate, that.bagDepositDate);
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
What is the rationale for this `hashCode` implementation?

Reply:
@janvanmansum auto-generated together with the equals() method, not sure if it's used to be honest. So let me check that first